source/SystemEditor.cpp

```diff
@@ -1090,11 +1090,13 @@ void SystemEditor::Randomize()
 	int planetCount = randPlanetCount(gen);
 	for(int i = 0; i < planetCount; ++i)
 	{
+		/*
 		constexpr int RANDOM_SPACE = 100;
 		int space = RANDOM_SPACE;
 		for(const auto &stellar : object->objects)
 			if(!stellar.isStar && stellar.parent == -1)
 				space += RANDOM_SPACE / 2;
+				*/
```

Comment:
I'm assuming this code doesn't compile?

Reply:
Yep, I can just delete the code instead, it looks like it's not doing anything. I have
```
	-Wall\
	-Werror\
```
on in the Makefile copying the SConstruct file, maybe it wasn't alerting for you on GCC?
```
source/SystemEditor.cpp:1094:7: error: variable 'space' set but not used [-Werror,-Wunused-but-set-variable]
                int space = RANDOM_SPACE;
                    ^
1 error generated.
```

Reply:
I'm not using gcc and seems like clang doesn't emit a warning here. I'll fix this